ChineseFilter/sections/adservers.txt

```diff
@@ -178,4 +178,5 @@
 ||f.zgbmw.com.cn^$third-party
 ||83.locksmaker.com.cn^$third-party
 ||yqrta.xyz^$third-party
-||zjgfq.com^$third-party
\ No newline at end of file
+||zjgfq.com^$third-party
+||exdynsrv.com^$third-party
```

Comment:
We have already this rule in the AdGuard Base.

Reply:
ok, marked the change